addons/point_of_sale/static/src/js/Popups/ClosePosPopup.js

```diff
@@ -128,14 +120,13 @@ export class ClosePosPopup extends AbstractAwaitablePopup {
         if (!this.closeSessionClicked) {
             this.closeSessionClicked = true;
 
-            if (this.env.pos.config.cashControl) {
+            if (this.cashControl) {
                 const response = await this.orm.call(
                     "pos.session",
-                    "post_closing_cash_details"[this.env.pos.pos_session.id],
+                    "post_closing_cash_details",
+                    [this.env.pos.pos_session.id],
                     {
-                        counted_cash: this.env.pos.config.cashControl
-                            ? this.state.payments[this.defaultCashDetails.id].counted
-                            : 0,
+                        counted_cash: this.cashControl ? this.state.payments[this.defaultCashDetails.id].counted : 0,
```

Comment:
unrelated to your changes but we are inside an if that already checks that cashControl is truthy, so this ternary is useless

Reply:
True.